app/managing-users.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: Managing users
+analytics_key: managing-users
+parent:
+  title: How to use the pilot service
+  url: /guide
+previous:
+  title: Reporting
+  url: /reporting
+next:
+  title: Managing vaccines
+  url: /managing-vaccines
+---
+
+{% include 'guide-contents.njk' %}
+
+## Adding new users  
+
+Only lead administrators can add and manage users. 
+
+To add a user, you must enter their: 
+
+* First name  
+* Last name 
+* NHS-approved email address  
+* Clinician status
+
+## Selecting clinician status 
+
+Select ‘yes’ if users are registered clinicians. This will allow them to assess patients and record their consent.  
+
+## Selecting permission levels 
+
+Choose and select a permission level for new users: 
+
+| **Permission level**               | **Lead administrator**    | **Administrator**    | **Recorder**    |
+|------------------------------------|---------------------------|----------------------|-----------------|
+| **Records and edits vaccinations** | Yes                       | Yes                  | Yes             |
+| **Adds and manages vaccines**      | Yes                       | Yes                  | No              |
+| **Creates reports**                | Yes                       | Yes                  | No              |
+| **Adds and manages users**         | Yes                       | No                   | No              |
```

Comment:
This confused me at first as I was expecting the permission levels to be below the "Permission level" heading rather than to the right... Might just be me though?

Could try flipping it around so that each permission level is a row, and the columns are the features they can and can't use?